src/Esfa.Vacancy.Domain/Validation/ErrorMessages.cs

```diff
@@ -63,6 +63,10 @@ public static class CreateApprenticeship
                 "'{PropertyName}' can't be specified when Location type is EmployerLocation or Nationwide";
 
             public const string FixedWageIsBelowApprenticeMinimumWage = "The fixed wage should not be less than the National Minimum Wage for Apprentices.";
+            public const string InvalidStandardLarsCode =
+                "'{PropertyName}' should be a number between 1 and 9999 when Training Type is Standard.";
+            public const string InvalidFrameworkLarsCode =
+                "'{PropertyName}' should be a in format '###-##-#' when Training Type is Framework.";
```

Comment:
`"'{PropertyName}' should be in the format '###-##-#' when Training Type is Framework.";` 💥

Reply:
fixed